provider.go

```diff
@@ -48,7 +48,10 @@ var (
 	regexAmzCredHeader = regexp.MustCompile(`X-Amz-(Credential|Security-Token|Signature)=[^&]*`)
 )
 
-var ErrInvalidTimecode = errors.New("invalid timecode")
+var (
+	ErrInvalidTimecode = errors.New("invalid timecode")
+	ErrWriteTimeout    = errors.New("write timeout")
```

Comment:
The error seems not to be currently used. Is this necessary?

Reply:
https://github.com/seqsense/kinesisvideomanager/blob/9035d1dfb54853f89c65d088a4bfe7137f238923/provider.go#L296-L298
should be `ErrWriteTimeout`